airflow/providers/google/cloud/links/cloud_sql.py

```diff
@@ -0,0 +1,69 @@
+#
```

Comment:
``CloudSQLExecuteQueryOperator`` will be part of another PR (in the future)? Or it'll be have assets.

Reply:
I will add it later, it is quite time consuming to analyze and set right now, so I will split it on two.

Reply:
Yeah, it also have separate example file so it's good idea to split it.